docs/maven-bnd.md

```diff
@@ -0,0 +1,109 @@
+# Migration to a Maven + Bnd based build system
+
+We are in the process of changing the build system.
+We are not accepting new pull requests that are still using the old way from here on.
+
+## Background
+
+openHAB is a java project that is build around the OSGi framework. openHAB addons are OSGi bundles.
```

Comment:
I suggest to change "addon" to "add-on" throughout the whole file.

Reply:
done